nss-day-cohort-47/automated-world-surprising-spuds
# Fixed Ireland

# Description
Fixed Ireland styling
Fixes # styling
## Type of change
# Testing Instructions
Fetch and serve dcu-ireland-styling, verify styling looks correct on ireland.html
# Checklist:
